### PR TITLE
[KEYCLOAK-9017] Add option to set database schema to container images

### PR DESCRIPTION
**Pull request by bartmentech**
This will fix https://issues.jboss.org/browse/KEYCLOAK-9017

### CODE REVIEW

**Comment by stianst** [comment]
Looks like this is only tested with PostgreSQL, we need to consider how it affects other DBs. The property you're setting on the connectionsJpa provider does affect all DBs afaik

File: docker-compose-examples/keycloak-postgres.yml [hunk 1]
```diff
@@ -6,20 +6,22 @@ volumes:
 
 services:
   postgres:
-      image: postgres
+      build: ./postgres
```
Comment (stianst) [inline docker-compose-examples/keycloak-postgres.yml]: Remove this change. We should not require users to build a Postgres image to use the example. I'd say the schema should be set to public here and it's up to the users to figure out how to add a schema to Postgres and not something we should cover on our end.

File: docker-compose-examples/postgres/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM postgres:11
```
Comment (stianst) [inline docker-compose-examples/postgres/Dockerfile]: Remove this file - see above comment

File: docker-compose-examples/postgres/create_schema.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (stianst) [inline docker-compose-examples/postgres/create_schema.sh]: Remove this file - see above comment

File: server/README.md [hunk 4]
```diff
@@ -72,6 +72,7 @@ Generic variable names can be used to configure any Database type, defaults may
 - `DB_ADDR`: Specify hostname of the database (optional)
 - `DB_PORT`: Specify port of the database (optional, default is DB vendor default port)
 - `DB_DATABASE`: Specify name of the database to use (optional, default is `keycloak`).
+- `DB_SCHEMA`: Specify name of the schema to use, this is only used when PostgreSQL is used (optional, default is `public`).
```
Comment (stianst) [inline server/README.md]: Change to:

Specify name of the schema to use for DB that support schemas (optional, default is `public` on Postgres).

**Comment by bartmentech** [comment]
Thank you for your comments. I will update it this week

**Comment by stianst** [comment]
Thanks

**Pull request merged**